test/parallel/test-tls-inception.js

```diff
@@ -12,10 +12,9 @@ var fs = require('fs');
 var path = require('path');
 var net = require('net');
 
-var options, a, b, portA, portB;
-var gotHello = false;
+var options, a, b;
 
-var body = new Buffer(4000).fill('A');
+var body = new Buffer(400000).fill('A');
```

Comment:
I've used that value because to reproduce the `ECONNRESET` error in `OS X` consistently.